tsconfig.json

```diff
@@ -3,13 +3,14 @@
     "declaration": true,
     "esModuleInterop": true,
     "importHelpers": true,
+    "lib": ["es2020"],
```

Comment:
Consider sticking with ES2019. v12 only supports ES2019 and is still an active LTS.

Reply:
I need ES2020 for `Promise.allSettled`